extensions/resource-deployment/src/services/notebookService.ts

```diff
@@ -59,9 +77,9 @@ export class NotebookService implements INotebookService {
 	 * @param title the title of the notebook
 	 * @param content the notebook content
 	 */
-	launchNotebookWithContent(title: string, content: string): Thenable<azdata.nb.NotebookEditor> {
+	async launchNotebookWithContent(title: string, content: string): Promise<azdata.nb.NotebookEditor> {
 		const uri: vscode.Uri = vscode.Uri.parse(`untitled:${title}`);
```

Comment:
This isn't awaited properly it looks like.

Reply:
fixed